algos/aho_corasick/render.js

```diff
@@ -0,0 +1,102 @@
+var d3 = window.d3
```

Comment:
Nit: Please use `let` instead of `var` unless you really need program scope.

Reply:
Done